app/routes.php

```diff
@@ -493,7 +495,7 @@ function ( string $value ) {
 		),
 		HttpKernelInterface::SUB_REQUEST
 	);
-} );
+} )->method( 'GET|POST' );
```

Comment:
just thought it would be nice to be able to use this route as the target of banner forms. we can, of course, use `donation/new` if you object this.